language/move-lang/tests/functional/vasps/dual_attestation.move

```diff
@@ -54,12 +54,13 @@ fun main() {
 //! sender: payer
 script {
 use 0x0::LBR;
+use 0x0::LCS;
 use 0x0::LibraAccount;
 fun main() {
-    let payment_id = x"0000000000000000000000000000000000000000000000000000000000000000";
-    let signature = x"62d6be393b8ec77fb2c12ff44ca8b5bd8bba83b805171bc99f0af3bdc619b20b8bd529452fe62dac022c80752af2af02fb610c20f01fb67a4d72789db2b8b703";
+    let ref_id = LCS::to_bytes(&7777);
+    let signature = x"8d83d481068a7b73a914e7d53f84cbfb8d55cdbd2306e17dae14fa0e6fa8ab0cea9f4c1bab22b701e57aa2e0e3f15bb6b40d62a32b7e158a384b6529f6463a09";
 
-    LibraAccount::pay_from_sender_with_metadata<LBR::T>({{payee}}, 1000, payment_id, signature);
+    LibraAccount::pay_from_sender_with_metadata<LBR::T>({{payee}}, 1000000, ref_id, signature);
```

Comment:
Wondering why this was changed to be greater than 1000 as opposed to exactly at the boundary?

Reply:
Ah, good question. 1000 is 1000 milliLBR, but the travel rule threshold should be 1000 * 1000 = 1000000 milliLBR. I haven't fixed the threshold (yet), but wanted to bump this test to the real threshold to avoid regenerating this hardcoded signature later (which is a bit painful...)